### PR TITLE
Added code to sort by descending order during publish, #PG-3142

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Added code to sort by descending order during publish
Fixes: #PG-3142

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by snake14 — COMMENTED** [review]
The code changes look good. During functional testing Tags, Triggers, and Variables are now sorted by descending modified date. My only concern is that it's grouping by type (Tags, Triggers, and Variables). So, you could have a bunch of tags that were updated months ago at the top and a Trigger or Variable that you just updated at the bottom of the list. I wonder if we should combine the types together so that everything is truly sorted by modified date?

**Comment by AltamashShaikh** [comment]
> The code changes look good. During functional testing Tags, Triggers, and Variables are now sorted by descending modified date. My only concern is that it's grouping by type (Tags, Triggers, and Variables). So, you could have a bunch of tags that were updated months ago at the top and a Trigger or Variable that you just updated at the bottom of the list. I wonder if we should combine the types together so that everything is truly sorted by modified date?

@snake14 Moved to client side sorting to solve the issue completely and removed all the extra code too

File: vue/src/Version/VersionEdit.vue [hunk 1]
```diff
@@ -85,31 +85,31 @@
             <h3>{{ translate('TagManager_ChangesSinceLastVersion') }}:</h3>
             <table v-content-table>
               <thead>
-                <tr>
```
Comment (snake14) [inline vue/src/Version/VersionEdit.vue]: It's definitely not a deal breaker, but shouldn't these lines be indented within the parent element? Same with the lines below?

**Review by snake14 — APPROVED** [review]
Nice work! The sort performs as expected. I just left an optional code style comment.

File: vue/src/Version/VersionEdit.vue (hunk 1 above)
Comment (AltamashShaikh) [reply vue/src/Version/VersionEdit.vue]: Removed the unwanted formatting

**Pull request merged**